themes/simplistick/simplistick.theme.bash

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+SCM_GIT_CHAR=""
+SCM_HG_CHAR=""
+SCM_SVN_CHAR=""
+SCM_NONE_CHAR=""
+SCM_THEME_PROMPT_DIRTY=" ${red}�"
+SCM_THEME_PROMPT_CLEAN=" ${bold_green}�"
+SCM_THEME_PROMPT_PREFIX="("
+SCM_THEME_PROMPT_SUFFIX="${green}) "
+SCM_GIT_AHEAD_CHAR="${green}�"
+SCM_GIT_BEHIND_CHAR="${red}�"
+
+GIT_THEME_PROMPT_DIRTY=" ${bold_red}�"
+GIT_THEME_PROMPT_CLEAN=" ${bold_green}�"
+GIT_THEME_PROMPT_PREFIX="${white}("
+GIT_THEME_PROMPT_SUFFIX="${white}) "
+
+RVM_THEME_PROMPT_PREFIX="("
+RVM_THEME_PROMPT_SUFFIX=") "
+
+VIRTUALENV_THEME_PROMPT_PREFIX="("
+VIRTUALENV_THEME_PROMPT_SUFFIX=") "
+
+RBENV_THEME_PROMPT_PREFIX="("
+RBENV_THEME_PROMPT_SUFFIX=") "
+
+RBFU_THEME_PROMPT_PREFIX="("
+RBFU_THEME_PROMPT_SUFFIX=") "
+
+function rvm_version_prompt {
+  if which rvm &> /dev/null; then
```

Comment:
Which is non-standard across unix systems, this should be:
```
if command -v rvm
```

Reply:
Bash-it provides the `_command_exists` function for this purpose:

```bash
if _command_exists rvm
```